chacra/async/__init__.py

```diff
@@ -107,6 +110,7 @@ def create_deb_repo(repo_id):
     # updated so we prevent piling up if other binaries are being posted
     repo.path = paths['absolute']
     repo.is_updating = True
+    repo.is_queued = False
```

Comment:
I don't think so, because ``needs_update=False`` and ``is_updating=True`` is set here also. I don't believe a new message would be added to the queue while this task is running.